### PR TITLE
Make alertmanager deployment more extensible

### PR DESCRIPTION
**Pull request by jenrik**
Add the ability to add extra arguments, containers, volumes, and volume mounts to the alertmanager deployment.

I implemented this change to able to add a sidecar to the alertmanager for automatic config reloading, and decided to round it out with the features from the server deployment.

### CODE REVIEW

File: charts/victoria-metrics-alert/README.md [hunk 1]
```diff
@@ -115,6 +115,11 @@ Change the values according to the need of the environment in ``victoria-metrics
 | alertmanager.configMap | string | `""` |  |
 | alertmanager.enabled | bool | `false` |  |
 | alertmanager.extraArgs | object | `{}` |  |
+| alertmanager.extraArgs | object | `{}` |  |
```
Comment (zekker6) [inline charts/victoria-metrics-alert/README.md:118]: It seems like `extraArgs` is now duplicated, could you please remove the duplicate?

**Review by zekker6 — COMMENTED** [review]
Hello @jenrik, thank you for the PR!

Please, could you check the comment?

Also, could you add an entry in a changelog about this change?
It is located here: https://github.com/VictoriaMetrics/helm-charts/blob/master/charts/victoria-metrics-alert/CHANGELOG.md
You need to replace `- TODO` entry with description of this change so that it will be included into change list of the release.

File: charts/victoria-metrics-alert/README.md (hunk 1 above)
Comment (jenrik) [reply charts/victoria-metrics-alert/README.md]: fix in fda37843f64e8b2179470067910ba4fdfadd34c5

**Comment by jenrik** [comment]
@zekker6 I fixed the duplicate value in the docs, and add an entry to the change log as requested

**Comment by jenrik** [comment]
I think the failed test might be flaky, I can't figure out a good reason for why it failed.

**Review by zekker6 — APPROVED** [review]
LGTM, @jenrik thank you for the contribution!

**Comment by zekker6** [comment]
FYI: @jenrik this PR was released in https://github.com/VictoriaMetrics/helm-charts/releases/tag/victoria-metrics-alert-0.9.0

**Pull request merged**